packages/remix-server-runtime/jsonify.ts

```diff
@@ -161,13 +167,27 @@ type _tests = [
   // object
   Expect<Equal<Pretty<Jsonify<{}>>, {}>>,
   Expect<Equal<Pretty<Jsonify<{a: string}>>, {a: string}>>,
+  Expect<Equal<Pretty<Jsonify<{a: Date}>>, {a: string}>>,
+  Expect<Equal<Pretty<Jsonify<{a: Map<unknown, unknown>}>>, {a: EmptyObject}>>,
+  Expect<Equal<Pretty<Jsonify<{a: Int8Array}>>, {a: Record<string, number>}>>,
+  Expect<Equal<Pretty<Jsonify<{a: [1, "two", Date, undefined, false]}>>, {a: [1, "two", string, null, false]}>>,
+  Expect<Equal<Pretty<Jsonify<{a: [unknown, 1]}>>, {a: [unknown, 1]}>>,
+  // @ts-expect-error
+  Expect<Equal<Pretty<Jsonify<{a: MyClass}>>, {a: {a: string}}>>,
```

Comment:
I can't seem to get these nested classes to work

@pcattori probably has a good idea where to look at